src/core/mac/sub_mac.cpp

```diff
@@ -248,6 +250,12 @@ void SubMac::HandleReceiveDone(RxFrame *aFrame, otError aError)
         UpdateFrameCounter(aFrame->mInfo.mRxInfo.mAckFrameCounter);
     }
 
+#if OPENTHREAD_CONFIG_MAC_CSL_DEBUG_ENABLE
+    otLogDebgMac("Received frame in CSL state %d, timestamp %u, target sample start time %u, time drift %d", mCslState,
+                 aFrame->mInfo.mRxInfo.mTimestamp, mCslSampleTime.GetValue(),
+                 aFrame->mInfo.mRxInfo.mTimestamp - mCslSampleTime.GetValue());
+#endif
```

Comment:
```suggestion
#if OPENTHREAD_CONFIG_MAC_CSL_DEBUG_ENABLE
    if (mState == kStateCslSample && aFrame != nullptr && aError == OT_ERROR_NONE)
    {
        otLogDebgMac("Received frame in CSL state %d, timestamp %u, target sample start time %u, time drift %d", mCslState,
                     aFrame->mInfo.mRxInfo.mTimestamp, mCslSampleTime.GetValue(),
                     aFrame->mInfo.mRxInfo.mTimestamp - mCslSampleTime.GetValue());
    }
#endif
```

Reply:
@brianljt 

I think I also hope to see the received frame when `mState` is not `kStateCslSample` so that I can know that other operations occupies CSL's time. The `nullptr` check is necessary. Thoughts?

Reply:
OK. Then you can add one more print field to show current `mState`. The `nullptr` check is still necessary.

Reply:
Suggestion updated.

Reply:
@brianljt , what is the plan with this suggested change in https://github.com/openthread/openthread/pull/5455#discussion_r477130387?

Reply:
@jwhui I'll update it soon for this suggestion.

Reply:
Updated.